### DISCUSSION TITLE
Hexadecane in Fluid List Missing

### DISCUSSION THREAD

**Discussion by Filippodaragona** [CoolProp/CoolProp#2330] [Q&A]
Hello,
I was not able to find Hexadecane in the Fluid List. 
Can I access it through CoolProp in some other way, or do I have to use another database entirely?
Thank you for the help,
Filippo

**Comment by ibell** [comment]
The EOS have been published. The fluid files are here: https://link.springer.com/article/10.1007/s10765-022-03059-0#Sec16
